### PR TITLE
Fix w3c false issues

### PR DESCRIPTION
**Pull request by alexpott**

### CODE REVIEW

**Comment by alexpott** [comment]
This is another fix at probably the correct level for https://www.drupal.org/project/drupal/issues/3458751#comment-15667563

File: test/Test/WebDriver/ChromeDriverNonW3CTest.php [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+use WebDriver\Session;
+
+/**
+ * ChromeDriver
+ *
+ * @package WebDriver
+ *
+ * @group Functional
+ */
+class ChromeDriverNonW3CTest extends WebDriverTestBase
```
Comment (longwave) [inline test/Test/WebDriver/ChromeDriverNonW3CTest.php]: Should this extend ChromeDriverTest instead? If we add a `protected $w3c` property then you can set that in the options from the constructor and also change the assertion to use it, similar to the other test.

Comment (alexpott) [reply test/Test/WebDriver/ChromeDriverNonW3CTest.php]: Fixed.

**Comment by longwave** [comment]
Closing and reopening in order to run the tests (see #11)

File: .ddev/docker-compose.selenium.yaml [hunk 2]
```diff
@@ -19,7 +19,23 @@ services:
       - ddev-global-cache:/mnt/ddev-global-cache
   chrome:
     container_name: ddev-${DDEV_SITENAME}-chrome
-    image: selenium/standalone-chrome:latest
+    image: selenium/standalone-chromium:latest
```
Comment (alexpott) [inline .ddev/docker-compose.selenium.yaml:22]: This change to the chrome image allows ddev to work on arm laptops. selenium/standalone-chrome:latest is not available on arm... whereas selenium/standalone-chrome:4.8 is.... go figure.

Comment (andriokha) [reply .ddev/docker-compose.selenium.yaml:22]: I suspect you might've already seen it, but [there's this fairly recent post about multi-arch images](https://www.selenium.dev/blog/2024/multi-arch-images-via-docker-selenium/). (Doesn't explain why it worked with 4.8, but it does speak as to why generally the `chrome` image doesn't support arm.)

**Review by chrfritsch — APPROVED** [review]
I've tested this patch on multiple of our sites with large behat test suites and it works fine. Thx everyone 👍🏼

**Review by longwave — APPROVED** [review]

**Pull request merged**